storybook/src/Safei/components/Organisms/PollOneImage/PollOneImage.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
Some Styles & functionality Issues As discussed in the components used before.

Reply:
styles adjusted, for functionality it would work fine, I know it works only when you click the radio check itself, not the whole tab, not going to invest time in this comp functionality at this point.🤝